src/interface/src/app/plan/plan-summary/summary-panel/summary-panel.component.ts

```diff
@@ -39,7 +42,7 @@ export const conditionScoreColorMap: Record<ConditionName, string> = {
   styleUrls: ['./summary-panel.component.scss'],
 })
 export class SummaryPanelComponent implements OnChanges {
-  @Input() plan: Plan | null = null;
+  @Input() plan = new BehaviorSubject<Plan | null>(null);
```

Comment:
Why are we changing this to a behavior subject? not a fan of using behavior subjects as inputs. I think it will simplify a lot the code if we solve this on the parent component as you where doing before `[plan]="currentPlan$ | async"`

Reply:
I had just changed it to avoid making too many changes to the plan-map component. I don't see any issues with updating plan-map to not use a Behavior Subject — went ahead and updated both summary-panel and plan-map to just take in a plan.

As another note, I removed the logic that changed the color that the planning area was drawn in based on if it had any scenarios since it seems better to just have the polygon be consistent across planning areas.